bin/convert-argv.js

```diff
@@ -80,12 +69,25 @@ module.exports = function(...args) {
 		const configArgList = Array.isArray(argv.config) ? argv.config : [argv.config];
 		configFiles = configArgList.map(mapConfigArg);
 	} else {
-		for (i = 0; i < defaultConfigFiles.length; i++) {
-			const webpackConfig = defaultConfigFiles[i].path;
-			if (fs.existsSync(webpackConfig)) {
+		const defaultConfigFileNames = ["webpack.config", "webpackfile"]
```

Comment:
This is a good chance to move `"webpack.config"` and `"webpackfile"` inside two constants